### DISCUSSION TITLE
Hardcore Death

### DISCUSSION THREAD

**Discussion by gregallen09** [chromiecraft/chromiecraft#7144] [General]
Hi. A half hour or so ago. I was in Felwood with Otric and Tillerson(dead). Otric was following Tillerson. Hopped off the road, Tillerson took over 2200 damage killing him. Otric took 750 damage. It wasnt a drop that should have killed any character. I only have screen shots of the damage they took. Can you please help

**Comment by gregallen09** [comment]
![WoWScrnShot_062324_134638](https://github.com/chromiecraft/chromiecraft/assets/173636751/d517898f-4bf9-4b1f-9768-28d57a09acae)
![WoWScrnShot_062324_134616](https://github.com/chromiecraft/chromiecraft/assets/173636751/5818e0da-bf87-4cb6-a0de-6dc8a80eb41a)
![WoWScrnShot_062324_134608](https://github.com/chromiecraft/chromiecraft/assets/173636751/f36f0ec0-fe6d-4daa-a279-a2651516cda3)

**Comment by andovcr** [comment]
Hi mate - do you have access to the ChromieCraft discord?  You would want to open a #support-ticket there to appeal any hardcore deaths.

**Comment by gregallen09** [comment]
Thank your responding. I'm having an issue verifying my acct for discord.
It keeps telling me my number is invalid. I tried my wife's and my sons.
All invalid. Someone suggested github, so posted there with screen shots.
You can see, the drop wasn't far enough to do over 2200 damage.

Greg

**Reply by andovcr** [reply]
Kk, if you can't access Discord, then try emailing <chromiecraft-support@googlegroups.com> (from [https://www.chromiecraft.com/en/privacy-policy-2/](https://www.chromiecraft.com/en/privacy-policy-2/)) to be connected with staff.  They are supposed to check that email, and I would be more confident getting a yes/no answer from there vs here (the suggestions forum).

If you don't hear back within a week, then I think it's fair to update this post with that info, so we can have someone else connect with staff on the best non-Discord alternative.

**Comment by gregallen09** [comment]
Thank you. I'll keep trying to get discord verified too.

**Comment by r-o-b-o-t-o** [comment]
Your character has been restored.
In the future, please try to record your gameplay to help with death appeals.

**Comment by gregallen09** [comment]
Thank you so much. /bow
